dashboard/app/helpers/levels_helper.rb

```diff
@@ -74,6 +74,8 @@ def set_channel
   end
 
   def select_and_track_autoplay_video
+    return if @level.try(:never_autoplay_video) == "true"
```

Comment:
Using `try` here because this levels_helper.rb code applies to all levels, not just the Blockly levels that have this option.